PathOfBuildingCommunity/PathOfBuilding
# Fix Summon Skeletons enchant not appearing in filtered list

### Description of the problem being solved:
Summon Skeletons enchant wasn't appearing in the enchant list when filtered to only your skills
### Steps taken to verify a working solution:
- Add Summon Skeletons
- Add a helmet
- See that Summon Skeletons doesn't appear in the filtered skills list

### Link to a build that showcases this PR:
```bash
eNq1G2uT2jjy8_IrXFRd1d6HGTCvmUwxuwXzyFA1JCxMkrtPKWEL0I5sEVueCbm6_34tyQYDlpGxL6lKjNVvdbe6G9H_86dHrTcchIT5t3X7slm3sO8wl_jL2_qXl8eL6_qff9T6E8RXnxfDiFCx8kftt758tih-wxTw6hZHwRLzrwml9negtEY-X2Hmj9HfLPjI3Nv6J-bjujVHvkt48smhKAw_IQ_f1mcOINctFDrYd-927xWgh4g_Y84r5h8DFq0l2zeC38fMBZjRePJ5-pJiSvw0U5D5t_6Eog0OZhxxK4R_busDh5M3DKDA95l4hANFRCMgd1VvaFDAWmiJn1KwzROwwygI-T3y4PE0zmyNsbsFa-nAJgF-WCywlP8uIPxuhXzHgHxR2HFEOVlTgoMtvH3Z1WE8HRG3m1ryL4wjej-ZnRZEQTIDi38jfDWkYEEjugJ6tPQJx8bgE0ZC5heS2gj4LqIUQssMlnlz4hvqOEY-umPhznZ2Lw90ggMIPr6H0TyBMMMOg3hNo7RbBkyyUbXcnskCm0MWUiRGKCrNeXo8zEzhChM-T6ApJCYzyBnfpYGWnh7-kQa80jrcPf5pQG7kczOmaUC729VzfWNcHjRGEfzwNNlR_aAXYLLahMRBdIx-Ei_yIBW-oFe849K6utb70nLFfQh_Ha6tZ_tIAnwG2h2j7jloK8RCHV77Oi9YtmC9vLPPuRGwI98xi8AvfoBDHLyljsxeLsIU4kKcynOKDTF2LOLoMjnbFKsl9mN-GzN1njF2Vh-hzpkijs3y7xaqm29WAWtkVgGYYdauIUIBIwlEjZEuP-QhFTTTg4-D5Wa2Ipju9LH1R1QaPhHtDq1NUKWp0_hGJt9nWMhr0qgFzfINBa7Z4VBUpjcUpvOrvuJQ5lLgZs6JobgDBBebFrGTgP0tymRaDG0QeCzanWG9VidXBwVupEJyTqiGYIrdyNk7jT5oMYcUuh9TBUAsSgthDDhHzus9c5e4EJPiGLNovYZ8IXbdFE8cc1McklSVctEzgP4MLpuO3dwD0ZzBDtqYwfaQN-dygGKuizilCyizAzdmsW0-x5AUPEj4sm-FVnwX9drNgVbHqHWRgIb904S9g-QrMYUIi0FDNbOrYLSiBNj_tTGmvwduxODBd6NAhIIxj0MMIzanBx5q7XiIcTC_sDvN3mXzQ7fZ7tmdXk-HvD_NsC-vr7oVgB5NDlq9du-y07qG-vzqqvMhHy9jjnAEmTlHMOSSPVUogHw8YyioX1bEHAHnTByOYTMmDoYy7ZX-dvO6lweYX2hkouQXPUcombWgEXRB2cwLsxfiQVUThveIIyvEKHBWz5CKb-v11KdHROkcjmd4K7D6DTmAFU8jb80CLl_eIeqEkubIX0fc8uX01COh830eLRZiUAo0eSCnuw-Pjw93L6OvD7EYaZTwlVD63Y-8uZj7qf93GX2GZcVihdE8VI-39a8Ev88E1j3miFDIXw6jFK1D7N7WF4iGwJnAo4SZgb4Oz6EmoaC_TJJONq0dgJ7Sw08ccNAXil0nIFgr13b9hFCKofABsW06aiLH6AmpQusOhVz5g8ZSMh_qqYjprFYdsZiDCz6DqJZzvHrCEnyzxiHUvmRBHOH3-Vv-AtAKKscujgMnmrPJ2e-4XNTTkElbR0At6pFV4tVhx6s5VpVJX2tVtapHv8cO0uquFvXI2xTDfPnVRDaVLVQOpU9w9Agnh6AZECpKO-3OPlC8BdET_MxXOFD1opbSGHJUApIbOAGZR1wfximIHFvJI0ljIbGmR1UTF40OYi0nE-2dPBqDpmH0pFTnrk1keajTnHw6zU-lSUussbxazdE_mQpoVI-Xc0SXHYp26-N-Jy81QDOsTw1iUY-sQuuYBA8iQwoyDO7xAkMCy42DLUxOAuaRfw-5kucY1JCUOtAGb4y4qhMvRU0qmU1sZ6lCtFR-ybRbYYpyUlHW9AcDjErMH8_RNak_BXKCEOTsp5yqwIzStrV_woiKboTRcgSPvi8opSfjIfLde1GNl1RUFPPRGoglkn3OKut2XnZItd9ICu6-rHFCK4RK_CP2wuEGOqVHkcUPeickDRsXwVxeYdjDSdi5eIEiKt7_FSFK-Ea0C6m3z-rihc8Cb_uNE5BasXfhmoqiKLlCSMzPz2plQHlMTLBL7KWsEwsk1bTlu_ilvHUx2EktNZZyE9-hkYtHfjzPAWkIOApFcyGZuEcivmTZGvCAzpbNb32QJgb-SNkc0VaCIruQuLHa551aiV9OEYHdmr1CZ8FFUMY3U1rN-h5te4_2SFSnkecJ-goxPOYqGf5QhhvJSkxuQgw4QYHaxiX2xOoY2h8XGrnGiIORG8LSDUkGno5Y_Ujt7YG5tsR3SoOz-oqXaNLE2DmR3soW_8gqDbWxSiC58Y1k56U7Ky8Wjy8BxomzSkZ27CjwIX0_R6jcFJsbcuiK4wpO7L4PvSY8dK-v2-26xYFc6m6Q2BVnh66k-jJ9Vg6x4nwd3jQa7-_vl2vEV2yBfxKKLx3mNdaABDJdSC0vBNnGAP4MlwP5R6mXUOqr20JhI1YVZJdqCuXEwycGdaJYEy-TD1J10c3GzfeMB0KfX4x5_5a6ilCCzMjHaB27vFiLA7IdByF0OvcErBHI_J9sqgD8V9L496WHxEYWz0lCiEKsvr3_htEa_EW8FnxUlw6A6SidIuj0NjfWl0-jv7481Aa-Q-C4BIeAoKwNmY-tJwwnKK-p-uzG6rU68fMQhckNBjDvjdW87Fx3WjVVg1qqCL2x7NZVLV2WHmN1m9fd2jNGywg-TgkHp67FiebGajVr8S7cWMML-bcmbTXFP26sq3YNmk1KHCLW7dp_AuQv8Q3Q_K9y4RByHqLW73b3otX85z9E0gkwSOBaKhmnMQCoCVD7YEpVC5K8tadYFqsVesPW763mRUfQCMQ3M0DBU9nVEp2BHksI2N3jnAzGLdnI1_bAu800pGjWxTdCFvgaecWWanKtNQ6sOLVbcmBtqUmvtWGRJLOl6SDfWoG3WlAxWe8rAm15EFoLFlhd8GLhSuBnC4hQAbBRJkWHbGXEjJk7FRpaUk_xRWy3HrtbnDy0EK2TEO2TEJ2TEN2TEMmAUWbgbcyIw00XWPunHmVwDALGZA55RKYyORJTOJY9e0drazDfhKGwnHRtS3DmKv83E_lO0Tmk0TqDRgGUIYO6qQq5s_QvIvsQU14G_xHOjVerXVzyVpbk7Qp27gwrtv4fNDoV6NKrQI5eRXtT2r2LbK46JUuJkOHYvdISFAkNKHHfRItckmW7oqxgl5akU4E32hVERaegGxTes1IMs1y_CssVz8rdsmKXPUjahQRwN0l5WMJnsjewvO93S1Monnx2CLbZhpc3VLeaHWuV9bxeBWmiXUHQtUtGQKeiw7dsJNrV7Gu7cA7qFMYoq2pVVfQ57lPRbvfKhk9F212V9551_AlC52SBM_HK7sOU-MszysNymVO7c1XkzzOdqLQJ2kWtble03Z1qbN8tbYHC8V9RnBaRfOBFFB-X1mrOk0yw5UxVXoVi_oIsjy42YR97mxn5hSfi9wp8dxtqjF0SeRlXoQ5_T7pDkb8rbWSzGIVDFoYnYQ9_IXsSQf10NhNsQpGDV4y6ONhTVl2rTO5wXaV_jpEFL6aCybWrBKltgJNc5E5wuvko-7fgU_Il36zF-9dvHP72-H-OKlMG
```
### Before screenshot:
![image](https://user-images.githubusercontent.com/1209372/215346713-1c0ffb1d-bfde-4866-8cef-1554479bbe6f.png)

### After screenshot:
![image](https://user-images.githubusercontent.com/1209372/215346706-e3aa463d-9627-49ce-9916-e5d569690dfc.png)
